### ISSUE TITLE
Update sidereal time documentation to make clear how to get an approximate answer

### ISSUE THREAD

**Issue by mhvk** [astropy/astropy#3373]
On the mailing list, Joe Ninan noted that one could not get a sidereal time even if one gave a time in UT1 [1]. This is because the more recent precession models require both UT1 and TT, but it is possible to get the sidereal time using older IAU1982 and IAU1994 models. This is a reminder to document this, perhaps both in the `Time.sidereal_time` docstring and in the Time documentation. (Of course, this is related to the question of whether the UT1-UTC offset should be ignored if one is outside the IERS table; see #3274, #3105, etc.).

[1] https://groups.google.com/forum/#!msg/astropy-dev/N2Ug4RPU4DU/Gr5YNOANARUJ

**Comment by mwcraig** [comment]
In the astropy-dev post @eteq indicated that in v1.0 astropy the error message was more helpful but the snippet below still generates an `IndexError: (some) times are outside of range covered by IERS table.`

```python
>>> import astropy; print astropy.__version__
1.0.1
>>> from astropy.time import Time
>>> t = Time.now()
>>> t.sidereal_time('mean', -97.5)
IndexError                                Traceback (most recent call last)
...
IndexError: (some) times are outside of range covered by IERS table.
```

**Comment by taldcroft** [comment]
Summary:

- No action.  Would be good to do.

**Comment by algerdnazgul** [comment]
I made this changes to the text after arriving this message without downloading IERS table before. I know that this is not according this task, but i think that w must mention it too.

**Comment by souravsingh** [comment]
Is the issue still open? If so, I would like to work on it.

**Comment by mhvk** [comment]
Definitely still open, but see #4436 for some progress on the implementation side. It may be wise to hold off until that has settled down...

**Comment by mhvk** [comment]
Closing this, given the auto-downloading of IERS A implemented in #4436.
